src/Core/Managed/Shared/DataContracts/RequestTelemetry.cs

```diff
@@ -35,9 +35,6 @@ public RequestTelemetry()
             this.Data = new RequestData();
             this.context = new TelemetryContext(this.Data.properties, new Dictionary<string, string>());
             this.Id = Convert.ToBase64String(BitConverter.GetBytes(WeakConcurrentRandom.Instance.Next()));
```

Comment:
No
1. we never try to override it or need to check if it ws changed in customer's code
2. we need it for correlation on early stages